### PR TITLE
Finish Accessibility Project at Score 100

### PR DESCRIPTION
**Pull request by Andyveloper**
# Changes made
- Improved Accessibility from a score of 94 to a 100

## Accesibility Report

- Page Title - no issue found
- Image text alternatives - Images alt text were modified 
- Text Heading - h3 changed to h2 to fix heading order: This has been corrected.
- Color contrast - Fixed button contrast to 4.5:1 to improve readability for disabled people
- Resize - no issues found
- Interaction - No issue found
- Moving content - no issues found
- Multimedia - No issues found
- The basic structure of the page - No issues found

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -225,11 +225,11 @@ <h2 class="formTitle">I'm always interested in hearing about new projects, so if
...
-        <li><a href="https://www.linkedin.com/in/mauricio-cantillo-moreno-374576184/"><img src="./img/social/dark-icons/Linkedin.png" alt=""></a></li>
-        <li><a href="https://www.angellist.com/"><img src="./img/social/dark-icons/Angelist.png" alt=""></a></li>
-        <li><a href="https://www.twitter.com/MauroCantillo_"><img src="./img/social/dark-icons/Twitter.png" alt=""></a></li>
-        <li><a href="https://www.medium.com"><img src="./img/social/dark-icons/Medium.png" alt=""></a></li>
+        <li><a href="https://github.com/Zeraltz"><img src="./img/social/dark-icons/Github.png" alt="link-text"></a></li>
+        <li><a href="https://www.linkedin.com/in/mauricio-cantillo-moreno-374576184/"><img src="./img/social/dark-icons/Linkedin.png" alt="link-text"></a></li>
+        <li><a href="https://www.angellist.com/"><img src="./img/social/dark-icons/Angelist.png" alt="link-text"></a></li>
+        <li><a href="https://www.twitter.com/MauroCantillo_"><img src="./img/social/dark-icons/Twitter.png" alt="link-text"></a></li>
+        <li><a href="https://www.medium.com"><img src="./img/social/dark-icons/Medium.png" alt="link-text"></a></li>
```
Comment (keroloussamy) [inline index.html]: - [ ] Kindly change the `alt` attribute with an appropriate alternative text, that's useful for interacting or understanding.
Like for the GitHub icon change it to `GitHub account` for example.

- [ ] Kindly change all your alt attributes that don't have descriptive text.

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @Zeraltz, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._

- [ ] On the Pull request description, please write a detailed accessibility report about all the checks you made on your website.
You should have something similar to this:

```
Accessibility Report:

- Page Title - no issue found
- Image text alternatives - Images alt text were modified 
- Text Heading - There was no h1 on the page: This has been corrected.
- Color contrast - No issues found
- Resize - Converted px values to rem for text size accessibility reasons
- Interaction - No issue found
- Moving content - Disabled moving content from the page
- Multimedia - No issues found
- The basic structure of the page - No issues found
```




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Andyveloper) [reply index.html]: @keroloussamy I wrote the report with all the changes I made, please let me know what do you mean by this: 
- "On the Pull request description, please write a detailed accessibility report about all the checks you made on your website. You should have something similar to this"

That's exactly what I did, or maybe I'm not understanding what you mean, please explain to me.

The other changes I'll do them right away

Comment (keroloussamy) [reply index.html]: Hello @Zeraltz 
Just put the points that you have checked and if there is something that there's no issue with it write next to it `no issue found` as an example `Page Title - no issue found`.
And if there's something that you upgraded write next to it what you did. Like `Image text alternatives - Fixed link images missing alternate text`

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**